src/FlatSharp.Runtime/IO/SpanWriterExtensions.cs

```diff
@@ -61,8 +68,8 @@ public static class SpanWriterExtensions
         spanWriter.WriteUOffset(span, offset, vectorStartOffset);
         spanWriter.WriteInt(span, numberOfItems, vectorStartOffset);
 
+        var start = span.Slice(vectorStartOffset + sizeof(uint), numberOfItems * size);
```

Comment:
Is there a usage that fails without specifying the length?

Reply:
No -- this is just something I add so things fail violently if we write past the bounds of what we expect. More of a defense in depth measure.